### ISSUE TITLE
[ISSUE] Test of LoRA

### ISSUE THREAD

**Issue by csjfwang** [ecmwf/WeatherGenerator#1154]
### Describe the task. It can be a feature, documentation, etc.

I am opening this issue to discuss about the experiment setting to test LoRA (already implemented).

On the single stream (ERA5) setting, without any pretraining, training with `LoRA + Embedding + Pred Head` has already shown benefits compare to only train `Embedding + Pred Head` within 1 epoch. 

I think it might be better to use multiple streams as we did on last hackathon challenge II, and there are some ways in my mind:
1. ERA5 Split as multiple streams. 
2. Using multiple streams setting like era5_nppatms_synop.
3. Using ERA5 and CERRA as two streams. 

Steps:
1. Training a model with `forecast mode` for a few epochs (8 or 16).
2. Finetune it on a new stream, keep the majority of params frozen, and compare: 
    A. finetuning `Embedding + Pred Head` for 8 epochs.
    B. finetuning `LoRA + Embedding + Pred Head` for 8 epochs.
3. Plot the loss curve. 

Which way do you suggest? 

### Area

- [ ] datasets, data readers, data preparation and transfer
- [x] model
- [x] science
- [ ] infrastructure and engineering
- [ ] evaluation, export and visualization
- [ ] documentation
